TTS/tts/models/tacotron.py

```diff
@@ -37,37 +40,52 @@ def __init__(self,
                              forward_attn, trans_agent, forward_attn_mask,
                              location_attn, attn_K, separate_stopnet,
                              bidirectional_decoder, double_decoder_consistency,
-                             ddc_r, gst)
-        decoder_in_features = 512 if num_speakers > 1 else 256
-        encoder_in_features = 512 if num_speakers > 1 else 256
-        speaker_embedding_dim = 256
-        proj_speaker_dim = 80 if num_speakers > 1 else 0
-        # base model layers
+                             ddc_r, gst, gst_embedding_dim, gst_num_heads, gst_style_tokens)
+
```

Comment:
Is there any multi-speaker part can be places under tacotron_abstract.py?

Reply:
I made the changes